datasets/spatial-ucr.yaml

```diff
@@ -0,0 +1,29 @@
+Name: UCR Center for Geospatial Sciences Public Data
+Description: |
+  This bucket contains multiple datasets (as Quilt packages) created by the
+  Center for Geospatial Sciences (CGS) at the University of California-Riverside.
+  The data in this bucket contains the following:
+ 
+  1) Tabular and geographic data from the US Census
+  2) Land Cover imagery collected from [Multi-Resolution Land Characteristics Consortium](https://www.mrlc.gov/)
+  3) Road network data processed from OpenStreetMap
+Contact: Eli Knaap <knaap@ucr.edu>
+Documentation: https://spatialucr.github.io/geosnap-guide/home
+ManagedBy: "[UCR Center for Geospatial Sciences](https://spatial.ucr.edu)"
+UpdateFrequency: Annually
+Tags:
```

Comment:
We can only use the tags in tags.yaml; those may have changed. @knaaptime had four tags, this is three, please check my work.